### DISCUSSION TITLE
vmq-passwd

### DISCUSSION THREAD

**Discussion by mths1** [vernemq/vernemq#2094] [General]
Would anyone mind if a push a CR that either removes the fact that -c will overwrite existing files, or add a force option to do so? It happend to myself and a colleque of mine that we accidently overwrote (as in deleted) the password file cause the options are a little bit misleading:

 -c : create a new password file. This will overwrite existing files.
 -D : delete the username rather than adding/updating its password.
 -U : update a plain text password file to use hashed passwords.

after you have read the -D option (which is about users deletion), once you are my aged your brain automaticaly chooses -c for create  user... and booom...

Maybe it is just me, but I think it is dangerous that a password tool just overwrites existing credential files without warning. I'd say following the unix philosophy it shouldn't to that at all (this is was rm is for), or at least it should be extra hard to do so (as in a -force option)

**Comment by ioolkos** [comment]
@mths1 I see, yes, I think it's a good idea to protect the `-c` option. The way you describe it makes me think it must have bitten other users too.
